portfolio/.classpath

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These .classpath, .project and .settings files are IDE metadata. They add clutter to the code, and apparently they can also make it harder to work with this repo using a different IDE. Based on this, it's probably better not to include them in the repo - you can get Git to ignore them with a [.gitignore file](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files).

See a similar comment on Kathleen's PR for a bit more info.

Reply:
deleted files from the repo